### PR TITLE
Update nano to 2.9.8 (fixes).

### PR DESCRIPTION
**Pull request by sciurius**
_Motivation:_ This PR fixes the changelog and adds an SSS for nano.

### CODE REVIEW

**Comment by m4tt075** [comment]
The changelog fix is fine, but SSS and dsm-control file are no longer needed / wanted.

**Comment by sciurius** [comment]
I tried without SSS and got a template start-stop-script that "exit 3" for status, indicating "not running"...

File: spk/nano/Makefile [hunk 1]
```diff
@@ -11,11 +11,12 @@ DESCRIPTION_GER = nano ist ein einfacher Editor, der die Bearbeitung von Dateien
...
 DISPLAY_NAME = nano
 STARTABLE = no
-CHANGELOG = "Update to 2.9.5"
+CHANGELOG = "Update to 2.9.8."
 
 HOMEPAGE   = http://www.nano-editor.org
 LICENSE    = GPLv2
 
 SERVICE_SETUP = src/service-setup.sh
+SSS_SCRIPT    = src/dsm-control.sh
```
Comment (ymartin59) [inline spk/nano/Makefile]: Please discard this package specific script

**Comment by sciurius** [comment]
Should I fix the mk/spksrc.service.non-startable here or make a separate PR?

**Comment by ymartin59** [comment]
@sciurius You can fix it here. No problem

**Comment by sciurius** [comment]
Ok, done.

**Pull request merged**